pimd/pim_sock.c

```diff
@@ -48,7 +48,12 @@ int pim_socket_raw(int protocol)
 
 	frr_with_privs(&pimd_privs) {
 
+#if PIM_IPV == 4
 		fd = socket(AF_INET, SOCK_RAW, protocol);
```

Comment:
`PIM_AF` for simplicity to avoid the `#if`

Reply:
Ack